BlueprintUI/Sources/Element/Accessibility.swift

```diff
@@ -0,0 +1,202 @@
+import UIKit
+
+public struct Accessibility {} //  Namespace for collecting Blueprint models of UIKit Accessibility features.
+
+// MARK: - Accessibility Traits
+extension Accessibility {
+    public enum Trait {
```

Comment:
Can we leave a comment on this describing what a trait is? Maybe copy it from the apple docs?